secureconfig/securestring.py

```diff
@@ -6,18 +6,18 @@ class SecureString(str):
     '''When garbage collected, leaves behind only a string of zeroes.'''
 
     def __init__(self, anystring):
-        self._string = anystring
+        self._string = anystring.encode()
```

Comment:
this was necessary to get tests passing.

securestring is very clever but not actually used anywhere by SecureConfigParser, so I would personally just remove it from the main codebase and perhaps have it on a branch...